authentication/tests/services/test_user_create.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from authentication.models import ConfirmationOTP
+from authentication.services import create_user
+
+
+@pytest.mark.django_db
+def test_user_is_created_inactive() -> None:
```

Comment:
I'm not sure what the convention of testing is. What I mean: part of the logic is tested here, but other part is tested in `test_apis.py`